code/Company.php

```diff
@@ -370,4 +372,41 @@ public function scaffoldSearchField()
     {
         return DropdownField::create('CompanyID', 'Company', self::get()->map())->setEmptyString('');
     }
+
+    public function providePermissions()
+    {
+        return [
+            'COMPANY_EDIT' => [
```

Comment:
Since we're using this class in multiple admin sections it makes sense to keep these permissions.